spinta/backends/mongo.py

```diff
@@ -196,3 +184,21 @@ def wipe(context: Context, model: Model, backend: Mongo):
 def prepare(context: Context, type: Date, backend: Mongo, value: date) -> datetime:
     # prepares date values for Mongo store, they must be converted to datetime
     return datetime(value.year, value.month, value.day)
+
+
+@prepare.register()
+def prepare(context: Context, action: str, model: Model, backend: Mongo, value: dict) -> dict:
+    if action in ('getall', 'getone'):
+        # Mongo returns ID under, key `_id`, but to conform to the interface
+        # we must change `_id` to `id`
+        #
+        # TODO: this must be fixed/implemented in the spinta/types/store.py::get()
+        # just like it's done on spinta/types/store.py::push()
+        id = str(value.pop('_id'))
+        return {
+            'type': model.name,
+            'id': id,
+            **value,
+        }
+    elif action in ('insert', 'update'):
+        return value['id']
```

Comment:
`prepare` must be recursive, because nested properties might have backend specific values, that needs to be converted to python-native values, for example dates are good candidates for this conversion.